embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/metadata/EmbraceMetadataService.kt

```diff
@@ -501,11 +506,23 @@ internal class EmbraceMetadataService private constructor(
             if (appFramework == AppFramework.REACT_NATIVE) {
                 reactNativeBundleId = metadataBackgroundWorker.submit<String?> {
                     val lastKnownJsBundleUrl = preferencesService.javaScriptBundleURL
-                    computeReactNativeBundleId(
-                        context,
-                        lastKnownJsBundleUrl,
-                        buildInfo.buildId
-                    )
+                    val lastKnownCodePushJsBundleId = preferencesService.codePushJsBundleId
+                    if (!lastKnownJsBundleUrl.isNullOrEmpty() && !lastKnownCodePushJsBundleId.isNullOrEmpty()) {
```

Comment:
Or there every two bundle Ids for any given app? If we want to distinguish whether they are updatable, can we store a boolean instead to signify that, rather than having two IDs which can be confusing?